src/promptflow/promptflow/_utils/capability_list.json

```diff
@@ -0,0 +1,12 @@
+[
+    {
+        "name": "Activate",
+        "description": "Bypass node execution when the node does not meet activate condition.",
+        "components": [
+            "runtime",
+            "executor",
+            "pfs"
+        ],
+        "state": "Ready"
```

Comment:
Add a field like "available since" to note from which version started to support this capability?
(I don't know. Just asking)

Reply:
Extensions currently don't need a capability list, as they can get the pf version and display the corresponding capabilities based on the version. However, UX cannot get the pf version, so "available since" may not be useful at the moment.